### PR TITLE
Screens/sign up

### PR DESCRIPTION
**Pull request by longle23**
I have worked very hard to make this work, best regards.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -3,17 +3,21 @@ import { createNativeStackNavigator } from '@react-navigation/native-stack';
...
     return (
         <NavigationContainer>
             <Stack.Navigator
                 initialRouteName="Home"
-                screenOptions={{ headerShown: false }}
-            >
-                <Stack.Screen name="Home" component={LoginScreen} />
+                screenOptions={{ headerShown: false }}>
+
+                <Stack.Screen name="Home" component={SignUpScreen} />
```
Comment (mhxx307) [inline App.js]: Not like this. Home must be LoginScreen not SignUp Screen

File: src/screens/SignUp/index.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+import Checkbox from 'expo-checkbox';
+import { useState } from 'react';
+
+function SignUp() {
+
+    const navigation = useNavigation();
+
+    const [isSelected, setSelection] = useState(false);
+
```
Comment (mhxx307) [inline src/screens/SignUp/index.js:17]: if you have a style.js , you should style in styles.js

**Review by mhxx307 — COMMENTED** [review]
i don't think this hard like you talk. I want you  fix these problem early. Thanks, best regards.

**Review by mhxx307 — COMMENTED** [review]
Good job! I will merge this branch

**Comment by longle23** [comment]
It's OK

**Pull request merged**